enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -488,13 +488,52 @@ release-planning question.
 
 TBD.
 
+#### Automated and Manual Gateway Deployments
+
+OpenShift Service Mesh and Istio have a feature called [automated deployment](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment)
+that creates an Envoy deployment and service in the same namespace for each Gateway
+if the Gateway's `spec.addresses` field is left unset. This enables users to seamlessly
+create a "shard" per Gateway, much like how each IngressController object is a shard
+for routes. It is enabled via the `PILOT_ENABLE_GATEWAY_API_DEPLOYMENT_CONTROLLER`
+env variable.
+
+Conversely, with [manual deployments](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#manual-deployment),
+if the Gateway has the `spec.addresses` field set, then it must manually link
+to an [ingress gateway](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#configuring-ingress-using-a-gateway).
+The user needs to make their own ingress gateway  service and deployment in the same
+namespace to manually link to, or they need to use the default ingress gateway (if enabled).
+
+If the ServiceMeshControlPlane's `spec.gateways.ingress.enabled` field is set to `true`,
+Istio creates an `istio-ingressgateway` service, in the same namespace as the control plane,
+that is a ready-to-use proxy which gateways can be manually linked to. Istio [discourages](https://istio.io/latest/docs/setup/additional-setup/gateway/)
+the use of this ingress gateway as it couples the gateway to the control plane.
+
+_To summarize, there are four ways a user could link Gateways to Ingress Gateways:_
+1. Use Automated Deployments
+2. Use a Manual Deployment, manually create a new ingress gateway service and deployment, and link to this service
+3. Use a Manual Deployment and link to the existing `istio-ingressgateway` ingress gateway
+4. Use a Manual Deployment and link to a previously created Automated Deployment ingress gateway
+
+OpenShift will not inhibit or alter the functionality of Automated Deployments, except
+restricting creation of Gateways to specific users (_see the Automated Deployment section
+in Risks and Mitigations_).
+
+Nor will OpenShift inhibit or alter the functionality of Manual Deployments. Users
+are responsible for understanding and creating links to manual deployments when creating
+Gateways.
+
+The choice between these two features depends on the user's desired sharding scheme. Manual
+linking, being more expressive, can establish a many-to-one Gateway-to-Gateway-Deployment
+relationship, while Automated Deployments strictly establish a one-to-one
+Gateway-to-Gateway-Deployment relationship. Arguably, Automated Deployments are more portable
+among Gateway API implementations due to the fact Manually Deployments require linking an
```

Comment:
```suggestion
among Gateway API implementations due to the fact that Manual Deployments require linking an
```
Hm, is there a reason to capitalize "Automated Deployment" and "Manual Deployment"?

Reply:
I was thinking it was a proper noun, but on account istio [doesn't](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment) in their docs, we probably shouldn't